qualtran/bloqs/chemistry/pbc/first_quantization/projectile/select_uv.py

```diff
@@ -77,32 +78,22 @@ def signature(self) -> Signature:
     def pretty_name(self) -> str:
         return r'SEL UV'
 
-    def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
+    def build_call_graph(self, ssa: 'SympySymbolAllocator') -> 'BloqCountDictT':
+        cost = Counter['Bloq']()
         # C8 and C9, one of the registers of size num_bits_n so need to account for this.
-        cost_tc_p = (SignedIntegerToTwosComplement(self.num_bits_p), 3)
-        cost_tc_n = (SignedIntegerToTwosComplement(self.num_bits_n), 3)
+        cost[SignedIntegerToTwosComplement(self.num_bits_p)] += 3
+        cost[SignedIntegerToTwosComplement(self.num_bits_n)] += 3
         # Adding nu into p / q. Nu is one bit larger than p.
-        cost_add_p = (Add(QInt(self.num_bits_p + 1)), 3)
-        cost_add_n = (Add(QInt(self.num_bits_n + 1)), 3)
-        cost_ctrl_add_p = (Toffoli(), 3 * (self.num_bits_p + 1))
-        cost_ctrl_add_n = (Toffoli(), 3 * (self.num_bits_n + 1))
+        cost[Add(QInt(self.num_bits_p + 1))] += 3
+        cost[Add(QInt(self.num_bits_n + 1))] += 3
+        cost[Toffoli()] += 3 * (self.num_bits_p + 1)
+        cost[Toffoli()] += 3 * (self.num_bits_n + 1)
         # + 2 as these numbers are larger from addition of $\nu$
-        cost_inv_tc_p = (SignedIntegerToTwosComplement(self.num_bits_p + 2), 3)
-        cost_inv_tc_n = (SignedIntegerToTwosComplement(self.num_bits_n + 2), 3)
+        cost[SignedIntegerToTwosComplement(self.num_bits_p + 2)] += 3
+        cost[SignedIntegerToTwosComplement(self.num_bits_n + 2)] += 3
```

Comment:
i feel like there may be some information lost by removing those variable names? Not blocking, but maybe include in comments, eg the cost[Toffoli()] += 3 * self.num_bits_p is for `ctrl_add_p` which isn't otherwise obvious

Reply:
Done.